shopfloor_reception/services/reception.py

```diff
@@ -48,6 +48,14 @@ class Reception(Component):
     _usage = "reception"
     _description = __doc__
 
+    def _check_picking_status(self, pickings):
```

Comment:
We should keep the original signature here:
```suggestion
    def _check_picking_status(self, pickings, states=None):
```
or
```suggestion
    def _check_picking_status(self, pickings, states=("assigned",)):
```